swatch-contracts/src/main/java/com/redhat/swatch/contract/resource/CapacityResource.java

```diff
@@ -52,6 +53,6 @@ public CapacityReportByMetricId getCapacityReportByMetricId(
       ServiceLevelType sla,
       UsageType usage)
       throws ApiException, ProcessingException {
-    throw new UnsupportedOperationException("Not yet implemented");
+    throw new NotImplementedYet();
```

Comment:
@kahowell I think you added this endpoint a while ago and it's still not implemented. Do you know if we can fully remove it?

Reply:
We can remove it for now. The intent was to move the customer facing APIs for capacity/subscriptions here, but it's not particularly helpful to have a stub just sitting :-).

Reply:
This would imply to undo much more changes than I initially expected (see https://github.com/RedHatInsights/rhsm-subscriptions/commit/9d2cb2288349898c632ce017ffca22c2d9a23436). So, I think it's best to keep the stub as it is and if it no longer makes sense to have it, do it as part of another ticket. Do you want me to create this ticket or prever keeping the stub as it is?